source-code.md

```diff
@@ -28,3 +28,5 @@ Consider encrypting the contents of the repository.
 
 Teams are responsible for ensuring that their Git repositories
 are backed up to another location.
+
+GitHub organisations should have two-factor authorisation enforced.
```

Comment:
Suggest adding "for all users via the GitHub settings", to make it clear this isn't just a policy we tell individual users to follow.